### PR TITLE
ResStock-HPXML: Infiltration adjustment for SFA/MF units

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Adds the SFA/MF unit infiltration adjustment based on exposed wall area.

image

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: measures/ResStockArguments/measure.rb [hunk 1]
```diff
@@ -466,6 +466,64 @@ def run(model, runner, user_arguments)
...
+        end
+      elsif args['geometry_unit_type'] == HPXML::ResidentialTypeSFA
+        n_floors = 1.0
+        n_units_per_floor = n_units
+        has_rear_units = false
+      end
+
+      # Calculate exposed wall area ratio for the unit (unit exposed wall area
+      # divided by average unit exposed wall area)
+      if (n_units_per_floor == 1) || (n_units_per_floor == 2) || ((n_units_per_floor == 4) && has_rear_units) # No middle unit(s)
```
Comment (aspeake) [inline measures/ResStockArguments/measure.rb:494]: I think this logic should be updated to `if (n_units_per_floor <=2) || ((n_units_per_floor == 4) && has_rear_units)`, as non-integer units/floor are now valid as of this PR: https://github.com/NREL/resstock/pull/558. 

Same thing would need fixed on `sync-tsvs`/`develop`, I can do that.

**Review by aspeake — CHANGES_REQUESTED** [review]

File: measures/ResStockArguments/measure.rb (hunk 1 above)
Comment (shorowit) [reply measures/ResStockArguments/measure.rb:494]: Hmm, I am looking through that PR and not seeing where non-integer values are allowed. Can you more directly show me?

Comment (aspeake) [reply measures/ResStockArguments/measure.rb:494]: https://github.com/NREL/resstock/pull/558/files?file-filters%5B%5D=.tsv#diff-26529a8eddbe8c9d6be409cf357bead9406fbbc929779ee3e2107371f859d3be

`Geometry Building Number Units MF` is no longer dependent on `Geometry Stories`. I suppose non-integer values were always valid, but now we actually get them from the sampling.

Comment (shorowit) [reply measures/ResStockArguments/measure.rb:494]: Ah thanks, I was looking for some kind of integer -> double change. I'll make your proposed change.

**Review by aspeake — APPROVED** [review]

**Pull request merged**